native/cocos/platform/android/AndroidPlatform.h

```diff
@@ -58,9 +58,9 @@ class CC_DLL AndroidPlatform : public UniversalPlatform {
 
     int32_t getHeight() const;
 
-    void setAndroidApp(android_app *app);
-
 private:
+    bool _isInited{false};
```

Comment:
Does it will be initialized again?

Reply:
Yes, game launched at the same process will trigger it.